deegree-ogcapi-features/src/main/resources/META-INF/schemas/ogcapi/features/3.4.0/example.xml

```diff
@@ -35,4 +35,10 @@
     <MetadataURL format="text/html">http://localhost:8080/linkZuHTml/89362</MetadataURL>
   </Metadata>
 
+  <ConfigureCollection>
+  	<id>River</id>
+  	<FeatureStoreId>River</FeatureStoreId>
```

Comment:
I don't see why a feature store ID would need to be configured here.
This is the configuration file for a dataset, and the related feature store ID is already configured in the file (in this case `test`). I would expect in this file to have only configuration for the one dataset that is configured here, not for others.

Reply:
Even I had that understanding but from your previous comment 

>  So if there are configurations for multiple collections (also across multiple datasets) only one will be recorded here.

I thought that there might be chances that single file may contain collections of different dataset. 

Now corrected it.